data.lua

```diff
@@ -5,7 +5,7 @@ data:extend{
 		type = "custom-input",
 		name = "inventory-cleanup",
 		key_sequence = "SHIFT + C",
-		consuming = "all"
+		consuming = "none"
```

Comment:
not the maintainer, but according to https://forums.factorio.com/viewtopic.php?p=492101#p492101 it should be `game-only` not `none`

```suggestion
		consuming = "game-only"
```

Reply:
According to the [discussion](https://mods.factorio.com/mod/even-distribution/discussion/5eaac3df06db07000ef375cf) on the mod page, this setting consuming to game-only does not work